covid19-etl/etl/vac_tracker.py

```diff
@@ -0,0 +1,184 @@
+import csv
+import re
+from contextlib import closing
+
+import requests
+
+from etl import base
+from helper.metadata_helper import MetadataHelper
+
+
+# map from gen3 fields
+MAP_FIELDS = {
+    "id": ("submitter_id", str),
```

Comment:
we should generate our own `submitter_id` using the same format as in other ETLs

Reply:
note- the original id is a uuid so we can use it as submitter_id since there are no other unique values we can use to generate a submitter_id that makes more sense